packages/vite-node/src/client.ts

```diff
@@ -178,7 +178,7 @@ export class ViteNodeRunner {
     })
 
     // add 'use strict' since ESM enables it by default
-    const fn = vm.runInThisContext(`'use strict';async (${Object.keys(context).join(',')})=>{{${transformed}\n}}`, {
+    const fn = vm.runInThisContext(`'use strict';async (${Object.keys(context).join(',')})=>{{${transformed.replace(/^\#\!.*/, '')}\n}}`, {
```

Comment:
I think we should also patch coverage in `src/integrations/coverage.ts#L100`, otherwise it will be off

Reply:
I'm not even sure how source maps work with this, maybe @Demivan can help us?